pkg/framework/backoff.go

```diff
@@ -6,7 +6,7 @@ import (
 
 const (
 	LongMaxWait  = 40 * time.Minute
-	ShortMaxWait = 2 * time.Minute
+	ShortMaxWait = 4 * time.Minute
```

Comment:
Making this longer for chart-operator. Seems to be working - uncertain why we need more time now though.
See https://circleci.com/gh/giantswarm/chart-operator/tree/update-e2e-harness